controller/controller.js

```diff
@@ -115,14 +115,38 @@ module.exports = class Controller extends BaseController {
       fields,
       route,
       baseUrl: req.baseUrl,
+      skipToMain: this.getFirstFormItem(req.form.options.fields),
       title: this.getTitle(route, lookup, req.form.options.fields, res.locals),
+      header: this.getHeader(route, lookup, res.locals),
+      captionHeading: this.getCaptionHeading(route, lookup, res.locals),
+      subHeading: this.getSubHeading(route, lookup, res.locals),
       intro: this.getIntro(route, lookup, res.locals),
       backLink: this.getBackLink(req, res),
       nextPage: this.getNextStep(req, res),
       errorLength: this.getErrorLength(req, res)
     }, stepLocals);
   }
 
+  getFirstFormItem(fields) {
+    let firstFieldKey
+    if (_.size(fields)) {
+      firstFieldKey = Object.keys(fields)[0]
+    }
+    return firstFieldKey | 'main-content';
+  }
```

Comment:
In the nhs one it just sets to 'main-content'. 

https://gitlab.com/nhsbsa/Libraries/hof-govfrontend-v3/-/blob/master/controller/controller.js#L135

This was an attempt to fix it but I think it needs to be reworked and tested, keeps falling back to 'main-content' so doesn't serve it's purpose